### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by ruslansennov** [javaslang/javaslang.github.io#7]
What about the new logo after rebranding? At least, it needs to be renamed

**Comment by danieldietrich** [comment]
Yes, you are right. I'm on vacation until end of June and have the time to be creative... :-)

**Comment by davinkevin** [comment]
It would be good to have access to the logo in `svg` format in order to create a wallpaper(s) 😄

**Comment by pivovarit** [comment]
The new logo is there, I believe :) closing...
